R/score_model_out.R

```diff
@@ -122,7 +125,7 @@ score_model_out <- function(model_out_tbl, target_observations, metrics = NULL,
 #' Get scoring metrics
 #'
 #' @param forecast A scoringutils `forecast` object (see
-#' [scoringutils::as_forecast()] for details).
```

Comment:
scoringutils::as_forecast no longer exists; as_forecast_doc_template seems like a weird thing to be referring to, but it seemed to have the most useful/general documentation of the topic.